src/pybind/mgr/orchestrator/module.py

```diff
@@ -986,7 +986,7 @@ def _iscsi_add(self,
 
     @_cli_write_command(
         'orch',
-        "name=action,type=CephChoices,strings=start|stop|restart|redeploy|reconfig "
+        "name=action,type=CephChoices,strings=start|restart|redeploy|reconfig "
```

Comment:
are we sure? this would apply to _all_ services, not just OSDs ...

which then would force a user to issue a stop per daemon?
```
$ ceph orch daemon stop nfs.foo.node1
$ ceph orch daemon stop nfs.foo.node2
...
```

Reply:
My thoughts were, you really shouldn't run this for anything that has a ok-to-stop command, as they'd fail anyway.

Reply:
We should be able to forcefully restartd/redeploy any daemons though.

Reply:
yes. this only removes this for whole services.